src/components/breadcrumbs/_breadcrumbs_simplified.scss

```diff
@@ -0,0 +1,85 @@
+/*!
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+.ouiSimplifiedBreadcrumbs {
+  @include ouiFontSizeS;
+  margin-bottom: -$ouiSizeXS; /* 1 */
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  min-width: 0; // Ensure it shrinks if the window is narrow
+  margin-left: 0;
+}
+
+.ouiSimplifiedBreadcrumb {
+  display: inline-block;
+  color: $ouiLinkColor !important; // sass-lint:disable-line no-important
+  margin-right: $ouiBreadcrumbSpacing;
+}
+
+.ouiSimplifiedBreadcrumb--collapsed {
+  flex-shrink: 0;
+  color: $ouiBreadcrumbCollapsedLink !important; // sass-lint:disable-line no-important
+  vertical-align: top !important; // sass-lint:disable-line no-important
+}
+
+.ouiSimplifiedBreadcrumb__collapsedLink:hover {
+  color: $ouiBreadCrumbHoverColor !important; // sass-lint:disable-line no-important
+}
+
+.ouiSimplifiedBreadcrumbs--truncate {
+  white-space: nowrap;
+  flex-wrap: nowrap;
+
+  .ouiSimplifiedBreadcrumb:not(.ouiSimplifiedBreadcrumb--collapsed) {
+    max-width: 100%;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    vertical-align: top; // overflow hidden causes misalignment of links and slashes, this fixes that
+  }
+
+  .ouiSimplifiedBreadcrumbWrapper:not(.ouiSimplifiedBreadcrumbWrapper--collapsed) {
+    max-width: $ouiBreadcrumbTruncateWidth;
+    overflow: hidden;
+    text-overflow: ellipsis;
+
+    &.ouiSimplifiedBreadcrumbWrapper--last {
+      max-width: none;
+    }
+  }
+}
+
+.ouiSimplifiedBreadcrumb--truncate {
+  @include ouiTextTruncate;
+  max-width: 100%;
+  text-align: center;
+  vertical-align: top; // overflow hidden causes misalignment of links and slashes, this fixes that
+}
+
+.ouiSimplifiedBreadcrumbWrapper--truncate {
+  max-width: $ouiBreadcrumbTruncateWidth;
+}
+
+.ouiBreadcrumbSeparator {
+  flex-shrink: 0;
+  display: inline-block;
+  margin-right: $ouiBreadcrumbSpacing;
+  width: 2px;
+  height: $ouiSize;
+  transform: translateY(-1px) rotate(15deg);
```

Comment:
Is this a pre-existing implementation of the slashes?

Reply:
yes, taken from the code in oui 1.1